easybuild/easyconfigs/j/Java/README.md

```diff
@@ -0,0 +1,8 @@
+In order to use these configs you must first download Java from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html). The file should look like this: jdk-8u162-linux-x64.tar.gz    
+
+To install, you must past eb the sourcepath to find the download. There are a number of ways to do this assuming the file is in ~/EB_Downloads.  
```

Comment:
`you must past eb the sourcepath to find the download.` ->

```
you must configure EasyBuild to use the right `sourcepath` so it can find the manually downloaded file.
```